docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Unified"
+sidebar_position: 100
+hide_title: true
+---
+
+```mdx-code-block
+import Badges from '@site/src/components/Badges';
+import ThemedImage from '@theme/ThemedImage';
+import DocCardList from '@theme/DocCardList';
+
+```
+<Badges badgeType="dbt-package Release" pkg="unified"></Badges>
+
+# Snowplow Unified Package
+
+**The package source code can be found in the [snowplow/dbt-snowplow-unified repo](https://github.com/snowplow/dbt-snowplow-unified), and the docs for the [model design here](https://snowplow.github.io/dbt-snowplow-unified/#!/overview/snowplow_unified).**
+
```

Comment:
I'm not sure what license exactly this is going to be under or if it will be a public repo, so we may need to add the relevant header to these pages to call that out. @miike